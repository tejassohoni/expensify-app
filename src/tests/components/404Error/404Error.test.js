import React from "react";
import { shallow } from "enzyme";
import NotFoundError from "../../../Components/404Error/404-error.component";

test("Should render ExpenseListItem", () => {
  const wrapper = shallow(<NotFoundError />);
  expect(wrapper).toMatchSnapshot();
});
