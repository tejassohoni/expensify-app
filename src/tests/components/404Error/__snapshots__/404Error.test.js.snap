// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseListItem 1`] = `
<div>
  404 Not Found - 
  <[object Object]
    to="/"
  >
    Go Home
  </[object Object]>
</div>
`;
